ctfo_server.h

```diff
@@ -101,15 +101,26 @@ struct CTFOHypermedia
 
 CURRENT_STRUCT(CTFOServerParams) {
   CURRENT_FIELD(api_port, int);
+  CURRENT_FIELD_DESCRIPTION(api_port, "Port to spawn CTFO API on.");
   CURRENT_FIELD(rest_port, int, 0);
+  CURRENT_FIELD_DESCRIPTION(rest_port, "Port to spawn RESTful server on (the same as api_port by default).");
   CURRENT_FIELD(midichlorians_port, int, 0);
```

Comment:
However, I'd suggest to pre-define separate constants for all ports and and add them as default values to the config structure. The main reason is that our HTTP server has one-thread-per-port model and it's better to separate different HTTP requests to avoid blocking.